### PR TITLE
ATO-871: Create a basic SPOT stub

### PR DESCRIPTION
**Pull request by ethanmills**
Currently just responds with a hard-coded identity. This will need to change over time. This also does no validation of the input and does not serve to test that the integration with SPOT is fully functional. It simply allows us to exercise the IPV Core stub.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -24,6 +24,9 @@
     "fix:prettier": "prettier --write \"**/*.{ts,json}\"",
     "test": "jest --passWithNoTests",
     "test:coverage": "jest --coverage --passWithNoTests",
+    "test:integration": "npm run localstack:up && (jest --config=./jest.config.integration.ts && e=0) || e=1 && npm run localstack:down && exit $e",
```
Comment (ethanmills) [inline package.json:27]: I need to follow up with github actions for this

File: spot-stub-template.yml [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+        Request:
+          Type: SQS
+          Properties:
+            BatchSize: 1
+            Enabled: true
+            Queue: !FindInMap [
+              EnvironmentConfiguration,
+              !Ref Environment,
+              requestQueueArn,
+            ]
```
Comment (Ryan-Andrews99) [inline spot-stub-template.yml:60]: Does this need a `FunctionResponseTypes` of `BatchItemFailures` for batchItemFailures to work correctly?

**Review by Ryan-Andrews99 — COMMENTED** [review]
One small comment, but overall looks good

File: spot-stub-template.yml (hunk 2 above)
Comment (ethanmills) [reply spot-stub-template.yml:60]: Good spot

**Review by Ryan-Andrews99 — APPROVED** [review]

**Pull request merged**